server.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express');
+const db = require('./data/db');
+
+const port = 5555;
+const server = express();
+server.use(express.json());
+
+const sendUserError = (status, message, res) => {
+  res.status(status).json({ errorMessage: message });
+  return;
+};
+
+server.get('/:id', (req, res) => {
+  const { id } = req.params;
+  console.log(id);
+  // 1st arg: route where a resource can be interacted with
+  // 2nd arg: callback to deal with sending responses, and handling incoming data.
+  res.send('Hello from express');
+});
+
+server.post('/api/users', (req, res) => {
+  const { name, bio, created_at, updated_at } = req.body;
```

Comment:
created_at and updated_at are handled inside the database, so remove them from here.